### ISSUE TITLE
A tracklist 

### ISSUE THREAD

**Issue by Lucas-mother3** [devos50/qemu#110]
A good addition to the readme file is the addition of a checklist, to track which components are emulated and/or planned to be emulated with this project.

**Comment by devos50** [comment]
Certainly! I think having a table with "full support", "partial support" or "no support" would be helpful, both for the IT1G and IT2G 👍

**Comment by Lucas-mother3** [comment]
> Certainly! I think having a table with "full support", "partial support" or "no support" would be helpful, both for the IT1G and IT2G 👍

A bit off topic here but why the iPod Touch specifically, not the first gen iPhone (also known as iPhone 2G)? Is any there a plan or is it a bit harder to emulate?

**Comment by devos50** [comment]
> A bit off topic here but why the iPod Touch specifically, not the first gen iPhone (also known as iPhone 2G)? Is any there a plan or is it a bit harder to emulate?

This is mostly because of simplicity. The iPhone has additional communication modules and hardware components which might have made emulation more difficult. Therefore, I opted for emulating the simplest device.

In hindsight, I don't think it should be _that_ difficult to emulate the iPhone as well, since its chipset is mostly similar to the iPod Touch 1G.
